02-tamagotchi-interaction/tests/interaction.rs

```diff
@@ -30,5 +28,19 @@ fn interaction_test() {
     sys.init_logger();
     let _program = Program::current(&sys);
 
-    // TODO: 6️⃣ Test new functionality
+    let tamagotchi_name = "Kaori";
```

Comment:
It would be great to check the mood in tests after several blocks (see the [`gtest::System::spend_blocks`](https://docs.gear.rs/gtest/struct.System.html#method.spend_blocks) function).